src/platforms/python/common/performance/index.mdx

```diff
@@ -3,7 +3,36 @@ title: Performance Monitoring
 description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
 ---
 
-Performance Monitoring helps you see everything from macro-level metrics to micro-level spans, and you’ll be able to cross-reference transactions with related issues, customize queries based on your personal needs, and substantially more and is avaliable for the Sentry Python SDK version ≥ 0.11.2.
+Sentry allows you to monitor the performance of your application, showing you how latency in one service may affect another service, and letting you pinpoint exactly which parts of a given operation may be responsible. To do this, it captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services, respectively. You can learn more about this model in our [Distributed Tracing](/product/performance/distributed-tracing/) docs. Performance Monitoring is available for the Sentry Python SDK version ≥ 0.11.2.
+
+## Enabling Tracing
+
+To get started automatically sending traces you can set the `traces_sample_rate` in your SDK config. This should be a number between `0` and `1`. (For example, to send 20% of transactions, set traces_sample_rate to `0.2`.)
+
+```python
+import sentry_sdk
+
+sentry_sdk.init(
+    "___PUBLIC_DSN___",
+
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production.
+    traces_sample_rate=1.0,
+)
+```
+
+Your application will now start automatically capturing transactions.
+
+<Alert level="info" title="Note">
+
+When you first enable tracing, the easiest thing to do to test it is to set `traces_sample_rate` to `1.0`, because that guarantees that every transaction will be sent to Sentry for you to see.
+
+Once you're done with testing, however, we recommend that you either **lower your `traces_sample_rate` value, or switch to using `traces_sampler` to dynamically sample and filter your transactions**.
+
+Without sampling, our atomatic instrumenation will send a transaction any time any user loads any page or navigates anywhere in your app. That's a lot of transactions! Sampling allows you to get representative data without overwhelming either your system or your Sentry transaction quota.
```

Comment:
We generally avoid exclamation points :) 
```suggestion
Without sampling, our automatic instrumenation will send a transaction any time any user loads any page or navigates anywhere in your app. Sampling allows you to send representative data without overwhelming either your system or your Sentry transaction quota.
```

Reply:
I'd also suggest changing this to "will send a transaction with every request to your server" or some such - since it's not just pageloads and navigations that generate requests, and not just webapps that servers serve.